File: 0027-remove-element/0027-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        l = len(nums)

        if l < 1: return 0

        p = 0
        for i in range(len(nums)):
            if nums[i] != val:
                nums[p] = nums[i]
                p += 1

        return p

File: 0027-remove-element/0027-remove-element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int l = nums.length;

        if(l<1) return 0;

        int p = 0;
        for(int i=0; i<l; i++)
        {
            if(nums[i]!=val){
                nums[p] = nums[i];
                p += 1;
            }
        }

        return p;
    }
}